apps/dotcom/client/src/routes.tsx

```diff
@@ -104,12 +110,7 @@ export const router = createRoutesFromElements(
 					lazy={() => import('./tla/pages/publish')}
 				/>
 				{/* Views that require login */}
-				<Route lazy={() => import('./tla/providers/RequireSignedInUser')}>
-					{/* User settings */}
-					<Route path={`/${PREFIX.tla}/profile`} lazy={() => import('./tla/pages/profile')} />
-					{/* Internal */}
-					<Route path={`/${PREFIX.tla}/debug`} lazy={() => import('./tla/pages/debug')} />
-				</Route>
+				<Route lazy={() => import('./tla/providers/RequireSignedInUser')}></Route>
```

Comment:
There's nothing inside here, do we want to keep it for any future routes?

Reply:
yeah, i'm just keeping it around as a stub for future routes